src/app/clusters/level-control/level-control.cpp

```diff
@@ -483,11 +485,11 @@ static void moveToLevelHandler(uint8_t commandId, uint8_t level, uint16_t transi
 
     // Move To Level commands cause the device to move from its current level to
     // the specified level at the specified rate.
-    if (MAX_LEVEL < level)
+    if (MAX_LEVEL <= level)
```

Comment:
Why this change and the one for the "min" case?  The net effect seems to be the same as the previous code...